### PR TITLE
Images plus Ray update

### PR DESCRIPTION
**Pull request by janodev**
Images section.
Manual update to changes in Ray Wenderlich 2019. Too many conflicts to be merged automatically.

### CODE REVIEW

**Review by rkreutz-teamwork — APPROVED** [review]

File: README.markdown [hunk 1]
```diff
@@ -593,12 +631,14 @@ func reticulateSplines(spline: [Double]) -> Bool {
 }
 ```
 
-For functions with long signatures, add line breaks at appropriate points and add an extra indent on subsequent lines:
+For functions with long signatures, put each parameter on a new line and add an extra indent on subsequent lines:
```
Comment (denisq1991) [inline README.markdown:634]: "long signature" is a bit ambiguous don't you think? We normally break a line for each parameter if the method signature includes more than two.

File: README.markdown [hunk 2]
```diff
@@ -593,12 +631,14 @@ func reticulateSplines(spline: [Double]) -> Bool {
...
 
 ```swift
-func reticulateSplines(spline: [Double], adjustmentFactor: Double,
-    translateConstant: Int, comment: String) -> Bool {
-
+func reticulateSplines(
+  spline: [Double], 
+  adjustmentFactor: Double,
+  translateConstant: Int, comment: String
```
Comment (denisq1991) [inline README.markdown:640]: Should `comment: String` be on a new line?

**Review by denisq1991 — APPROVED** [review]
Good stuff, thanks for updating all this @j4n0

File: README.markdown (hunk 1 above)
Comment (dfreniche) [reply README.markdown:634]: @denisq1991 I think that's exactly what @j4n0 refers to here: you break down a long function signature using newlines as per example below. The end result: you never have long signatures _in the same line_

File: README.markdown (hunk 2 above)
Comment (dfreniche) [reply README.markdown:640]: @j4n0 yes, this should be in a new line

File: README.markdown [hunk 3]
```diff
@@ -1449,37 +1618,16 @@ expect(optionalValue?.date).to(equal(dateFormatter.date(from: "2018-06-20")!)) /
...
+
+```swift
+//
+//  Filename.swift
+//  Framework
+//
+//  Created by Jane Doe on 08/01/2019.
+//  Copyright © 2019 Teamwork. All rights reserved.
+//
 ```
```
Comment (dfreniche) [inline README.markdown:1631]: Superb

**Review by dfreniche — CHANGES_REQUESTED** [review]
Address the few comments and merge this thing away!

**Pull request merged**